charts/argo-cd/templates/argocd-server/service.yaml

```diff
@@ -3,6 +3,7 @@ kind: Service
 metadata:
   {{- if .Values.server.service.annotations }}
   annotations:
+    external-dns.alpha.kubernetes.io/hostname: {{ .Release.Name }}.{{ .Release.Namespace }}.coreweave.cloud
```

Comment:
Do we want it optional or just always on? Since we also do an ingress we could do one or the other?

Reply:
let's hold on a moment here I am going to PR an updated chart